dataprofiler/profilers/profile_builder.py

```diff
@@ -1078,7 +1079,15 @@ def load(cls, filepath: str) -> BaseProfiler:
         """
         # Load profile from disk
         with open(filepath, "rb") as infile:
-            data: dict = pickle.load(infile)
+
+            if filepath.endswith(".json"):
+                data: dict = json.load(infile)
+            elif filepath.endswith((".pickle", ".pkl")):
+                data = pickle.load(infile)
```

Comment:
i love the simplicity of this, but I'm not sure based on the `save()` implementation that this will account for some of the edge cases.... 🤔

Reply:
do we wanna do `load_method` to load similarly to how we did `save_method` for `save()`

Reply:
Also the json functionality should be using `json_decoder.load_profiler()` vs `json.load()` that way we can directly convert from json file to profiler object.

Reply:
unsure about the edge cases for load...

Reply:
@micdavis I don't see a `json_decoder.load_profiler()`

Reply:
`json_decoder.load_profiler(data, config={})` This is intentional for the config to use `{}` bc it allows for reuse of already loaded datalabelers

Reply:
after the `load_profiler` we can return with the output.

Reply:
it honestly might be best to separate this func into two for clarity, but could be a future refactor